vendor/vendor.json

```diff
@@ -7,6 +7,12 @@
 			"revision": ""
 		},
 		{
+			"checksumSHA1": "Vrt1uoeOOk9fp1aa9d7huQhzexU=",
+			"path": "code.gitea.io/sdk/gitea",
```

Comment:
Just to give you more background information: We introduced a custom import path directly at the beginning to be able to switch to dog-fooding at some point without distroying previous installs.